users/vieting/experiments/switchboard/ctc/feat/fullsum_ctc_raw_samples.py

```diff
@@ -167,6 +171,8 @@ def make_conformer_fullsum_ctc_model(
                 **specaug_old,
             }
             from_list, python_code = specaug_func(network, from_list=from_list, **specaug_old_args)
+        elif specaug_new is not None:
```

Comment:
Maybe add similar check as above:
```suggestion
        elif specaug_new is not None:
            assert specaug_old is None
```